### PR TITLE
Added instructions for creating sample applications

### PR DESCRIPTION
**Pull request by HarshCasper**
[RHDEVDOCS-2074](https://issues.redhat.com/browse/RHDEVDOCS-2074): Create applications easily based on available samples

Jira Task: https://issues.redhat.com/browse/RHDEVDOCS-2074

Developer Jira: https://issues.redhat.com/browse/ODC-4091

This PR has been reviewed by the SMEs and QEs.

Direct Preview Link: https://deploy-preview-30553--osdocs.netlify.app/openshift-enterprise/latest/applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.html#odc-creating-sample-applications_odc-creating-applications-using-developer-perspective

Versions: OpenShift 4.6 Async, OpenShift 4.7

### CODE REVIEW

File: modules/odc-creating-sample-applications.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+.Procedure
+
+. In the *+Add* view, click *Samples* to see the *Samples* page. 
+. On the *Samples* page, select one of the available sample applications to see the *Create Sample Application* form. 
+. In the *Create Sample Application Form*:
+* In the *Name* field, the deployment name is displayed by default. You can modify this name as required.
+* In the *Builder Image Version*, a builder image is selected by default. You can modify this image version by using the *Builder Image Version* drop-down list.
+* A sample Git repository URL is added by default.
+. Click *Create* to create the sample application. Once the resource is created and deployment is added to topology, you can check the build status of the resource.
```
Comment (christianvogt) [inline modules/odc-creating-sample-applications.adoc:21]: This line read oddly.

Comment (HarshCasper) [reply modules/odc-creating-sample-applications.adoc]: Fixed!

File: modules/odc-creating-sample-applications.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+
+. In the *+Add* view, click *Samples* to see the *Samples* page. 
+. On the *Samples* page, select one of the available sample applications to see the *Create Sample Application* form. 
+. In the *Create Sample Application Form*:
+* In the *Name* field, the deployment name is displayed by default. You can modify this name as required.
+* In the *Builder Image Version*, a builder image is selected by default. You can modify this image version by using the *Builder Image Version* drop-down list.
+* A sample Git repository URL is added by default.
+. Click *Create* to create the sample application. The build status of the sample application is displayed on the *Topology* view. After the sample application is created, you can see the deployment added to the application.
```
Comment (bmcelvee) [inline modules/odc-creating-sample-applications.adoc:20]: One suggestion (take it or leave it :smiley_cat:):  
. Click *Create* to create the sample application. 
+
The build status of the sample application is displayed on the *Topology* view. After the sample application is created, you can see the deployment added to the application.

**Comment by bmcelvee** [comment]
@HarshCasper LGTM! I had one optional suggestion. What labels does this PR need? And to which version(s) does it need to be cherry picked? Thanks!

File: modules/odc-creating-sample-applications.adoc (hunk 2 above)
Comment (HarshCasper) [reply modules/odc-creating-sample-applications.adoc:20]: Hi @bmcelvee 

Thanks for the suggestion! However in the docs, **Topology** is a UI Element and we have used **Create** in bolds as well. It is not intuitive enough to use italics for the same. Would love to know your thoughts about the same.

**Comment by HarshCasper** [comment]
> @HarshCasper LGTM! I had one optional suggestion. What labels does this PR need? And to which version(s) does it need to be cherry picked? Thanks!

I'm sorry to have missed them out. I have put the versions this PR needs to go into.

File: modules/odc-creating-sample-applications.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/odc-creating-sample-applications.adoc:20]: Oh how strange! Must have been a formatting quirk from Git - I was just suggesting adding a hard break following the procedural step, and nesting the build status portion underneath. Not a big deal though!

**Pull request merged**